static/downloadables/runPrsCLI.sh

```diff
@@ -215,6 +215,13 @@ learnAboutParameters () {
                 echo "the polygenic risk scores calculated. Alternativly, the path to a TXT file that"
                 echo "contains rsIDs in the format of 1 rsID per line, with the genotypes following"
                 echo "on the same line. (ex. rs6656401:AA or rs6656401:A)"
+                echo ""
+                echo "Additionally, VCF files separated into chromosomes are allowed, using Bash Expansion."
```

Comment:
I am not familiar with bash expansion. Are you saying that you still pass in one parameter with a '-f" before it? Could you put a short example right after this paragraph?
Ex: 'path/to/file1 path/to/file2'

An alternative that I think is more standard is to allow multiple -f options that all get added to a list in the option evaluation section.